setup.py

```diff
@@ -36,4 +38,4 @@
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
   ],
```

Comment:
I just tested these on Python 3.10 so I think it should work on later python versions :)